document-store/src/main/java/org/hypertrace/core/documentstore/Collection.java

```diff
@@ -183,4 +185,6 @@ CloseableIterator<Document> bulkUpsertAndReturnOlderDocuments(Map<Key, Document>
   UpdateResult update(Key key, Document document, Filter condition) throws IOException;
 
   String UNSUPPORTED_QUERY_OPERATION = "Query operation is not supported";
+
+    JsonNode getJsonNodeAtPath(String path1, JsonNode rootNode, boolean b);
```

Comment:
Could you share some context of why is this API needed?

Reply:
removed this. Not required